src/lib/Microsoft.Fx.Portability/Analysis/AnalysisEngine.cs

```diff
@@ -173,6 +208,20 @@ private MemberInfo ProcessMemberInfo(IApiCatalogLookup catalog, IEnumerable<Fram
             return member;
         }
 
+        /// <summary>
+        /// Identitifies the status of an API Exception for all of the targets.
+        /// </summary>
+        private static ExceptionInfo ProcessExceptionInfo(IApiCatalogLookup catalog, IEnumerable<FrameworkName> targets, MemberInfo member)
+        {
+            var exceptionHold = new ExceptionInfo();
+            exceptionHold.MemberDocId = member.MemberDocId;
+            exceptionHold.DefinedInAssemblyIdentity = member.DefinedInAssemblyIdentity;
+            exceptionHold.IsSupportedAcrossTargets = IsSupportedAcrossTargets(catalog, member.MemberDocId, targets, out var ignore);
+            exceptionHold.ExceptionsThrown = GetThrownExceptions(catalog, member.MemberDocId, targets);
```

Comment:
Maybe checking null of GetThrownExceptions(catalog, member.MemberDocId, targets) return value first before creating exceptionHold have better performance and less memory consumption?

Reply:
Great find, went ahead and changed it.